### PR TITLE
docs: improve README.md

### LINKED ISSUE #601: Improve documentation of GitHub repo
_CII Best Practice badge: The information on how to contribute SHOULD include the requirements for acceptable contributions (e.g., a reference to any required coding standard). (URL required)_

As a developer, I would like to know what problem MetabolicAtlas solves and how to contribute (including a reference to coding standard), so that it is straightforward for me to understand the project and contribute. 

Checklist
- [ ] describe in the readme what MetAtlas is about
- [ ] Link to the contribution guideline
- [ ] Update the CII badget 
- [ ] Update in the readme about how the users should report a bug or vulnerability

### PR DESCRIPTION
**Pull request by inghylt**
**Related issue(s) and PR(s)**  
This PR closes #601

**Type of change**  

- [x] Other, improve README.md
 
**List of changes made**  

Added sections "About Metabolic Atlas" and "Contributing"

**Further comments**  

Just a thought, does it make more sense to have CONTRIBUTING.md in the root of the project? To me it is not super intuitive to have in the .github folder

**Checklist**  

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by nanjiangshu — APPROVED** [review]
Very well written @inghylt!

**Comment by nanjiangshu** [comment]
A comment might be related to this issue is that the [contribution guidelines](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/.github/CONTRIBUTING.md#creating-a-new-issue) might be a bit outdated for submitting an issue, since we already templates. So probably better just ask the issue submitter to choose a proper template instead of write the issue in the specified format.

**Comment by inghylt** [comment]
@nanjiangshu Nice suggestion! Tried to incorporate it in d2157b9054739d05251ff3ecaeaff257f4bf7808

File: README.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ If you use *Metabolic Atlas* in your scientific work, please cite:
 >
 > Robinson, J., et al, 2020. _An atlas of human metabolism_. Science Signaling 13 [doi:10.1126/scisignal.aaz1482 ](https://doi.org/10.1126/scisignal.aaz1482)
 
+## About Metabolic Atlas
+Metabolic Atlas is a web platform integrating open-source genome scale metabolic models (GEMs) for easy browsing and analysis. The goal is to collect many curated GEMs, and to bring these models closer to [FAIR principles](https://en.wikipedia.org/wiki/FAIR_data). The webiste provides visualisations and comparisons of the GEMs, and links to resourcess, algorithms, other databases, and more general software applications. In short, the vision is to create a one-stop-shop for everything metabolism related.
```
Comment (mihai-sysbio) [inline README.md:11]:
```suggestion
Metabolic Atlas is a web platform integrating open-source genome scale metabolic models (GEMs) for easy browsing and analysis. The goal is to collect many curated GEMs, and to bring these models closer to [FAIR principles](https://en.wikipedia.org/wiki/FAIR_data). The website provides visualisations and comparisons of the GEMs, and links to resourcess, algorithms, other databases, and more general software applications. In short, the vision is to create a one-stop-shop for everything metabolism related. 
```

**Review by mihai-sysbio — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ If you use *Metabolic Atlas* in your scientific work, please cite:
 >
 > Robinson, J., et al, 2020. _An atlas of human metabolism_. Science Signaling 13 [doi:10.1126/scisignal.aaz1482 ](https://doi.org/10.1126/scisignal.aaz1482)
 
+## About Metabolic Atlas
+Metabolic Atlas is a web platform integrating open-source genome scale metabolic models (GEMs) for easy browsing and analysis. The goal is to collect many curated GEMs, and to bring these models closer to [FAIR principles](https://en.wikipedia.org/wiki/FAIR_data). The webiste provides visualisations and comparisons of the GEMs, and links to resourcess, algorithms, other databases, and more general software applications. In short, the vision is to create a one-stop-shop for everything metabolism related. 
+
+## Contributing
+If you would like to contribute to the project, for example if you have ideas for new features, discovered a bug or if you would like to improve the code base, please have a look at [CONTRIBUTING.md](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/.github/CONTRIBUTING.md).
```
Comment (mihai-sysbio) [inline README.md:14]: I would like this section to be even more encouraging. I've added a suggestion at the end. If you have any phrases/ideas to encourage contributions, maybe from other projects, please adopt them.
```suggestion
If you would like to contribute to the project, for example if you have ideas for new features, discovered a bug or if you would like to improve the code base, please have a look at [CONTRIBUTING.md](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/.github/CONTRIBUTING.md). We are looking forward to your ideas and contributions.
```

Comment (inghylt) [reply README.md:14]: I'm not really sure about this, "we" is not something used elsewhere in the text (or repo in general?). Maybe something like "All ideas and contributions are highly appreciated" or similar.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ In this guide you will get overview of the contribution workflow from opening an
 
 ## Creating a new issue
 
-If you spot a problem with the website or have any kind of suggestions, search if an issue already exists. If a related issue doesn't exist, please create a new issue with a description in the following template.
+If you spot a problem with the website or have any kind of suggestions, search if an issue already exists. If a related issue doesn't exist, please create a new issue. Use a suitable issue template, e.g. bug report or feature request, or create a blank issue with a description as following:
```
Comment (MalinAhlberg) [inline .github/CONTRIBUTING.md:9]: If we want to be even more helpful to unexperienced users, we could add the link:
```suggestion
If you spot a problem with the website or have any kind of suggestions, search if an issue already exists. If a related issue doesn't exist, please [create a new issue](https://github.com/MetabolicAtlas/MetabolicAtlas/issues/new/choose). Use a suitable issue template, e.g. bug report or feature request, or create a blank issue with a description as following:
```

**Review by MalinAhlberg — APPROVED** [review]
Very nice work and nice text!

I support the idea of moving the `CONTRIBUTION.md` to the root folder, to make it more visible.

**Review by e0 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (e0) [reply README.md:14]: > I'm not really sure about this, "we" is not something used elsewhere in the text (or repo in general?). Maybe something like "All ideas and contributions are highly appreciated" or similar.

I prefer more neutral tones so I think it's fine without any addition, but this is a very personal preference. If we are to add something I like your suggestion of the third-person narrative.

Comment (inghylt) [reply README.md]: Added a sentence  in ea44798bb04f5a20bde19f324630bde1f862693a

**Comment by inghylt** [comment]
Moved CONTRIBUTING.md as discussed during the meeting. Also looked at some popular os projects (Flutter, Docker, tensorflow, kubernetes) and they all have it in root

**Comment by inghylt** [comment]
@mihai-sysbio I am ready to merge if you feel like your comment is solved

File: README.md (hunk 2 above)
Comment (mihai-sysbio) [reply README.md]: Looks great!

**Comment by mihai-sysbio** [comment]
Thanks for the ping @inghylt, I have now resolved that comment.

**Pull request merged**